docs/samples/hub-state-import-data-sample.json

```diff
@@ -611,312 +206,375 @@
+        },
+        "properties": {
+          "desired": {
+            "prop2": 5
+          }
+        },
+        "capabilities": {
+          "iotEdge": true
+        },
+        "symmetricKey": {
+          "primaryKey": "mock_key",
+          "secondaryKey": "mock_key"
+        }
+      },
+      {
+        "deviceId": "device2",
+        "status": "enabled",
+        "connectionState": "Disconnected",
+        "authenticationType": "selfSigned",
+        "x509Thumbprint": {
+          "primaryThumbprint": "085836B09D1272B03F21FDFA8DC055D658C5AF14",
+          "secondaryThumbprint": null
+        },
+        "tags": {
+          "deviceType": "Type1, Type2, Type3",
+          "example": "val"
+        },
+        "properties": {
+          "desired": {
+            "prop2": 5
+          }
+        },
+        "capabilities": {
+          "iotEdge": true
+        }
+      },
+      {
+        "deviceId": "device2-child",
+        "status": "enabled",
+        "connectionState": "Disconnected",
+        "authenticationType": "sas",
+        "x509Thumbprint": {
+          "primaryThumbprint": null,
+          "secondaryThumbprint": null
+        },
+        "properties": {
+          "desired": {
+            "prop2": 5
+          }
+        },
+        "capabilities": {
+          "iotEdge": false
+        },
+        "symmetricKey": {
+          "primaryKey": "mock_key",
+          "secondaryKey": "mock_key"
+        }
+      },
+      {
+        "deviceId": "simDevice",
+        "status": "enabled",
+        "connectionState": "Disconnected",
+        "authenticationType": "sas",
+        "x509Thumbprint": {
+          "primaryThumbprint": null,
+          "secondaryThumbprint": null
+        },
+        "tags": {
+          "deviceType": "Type1, Type2, Type3"
+        },
+        "properties": {
+          "desired": {
+            "property1": 24,
+            "prop2": 5
+          }
+        },
+        "capabilities": {
+          "iotEdge": false
+        },
+        "symmetricKey": {
+          "primaryKey": "mock_key",
+          "secondaryKey": "mock_key"
+        }
+      },
+      {
+        "deviceId": "device3",
+        "status": "enabled",
+        "connectionState": "Disconnected",
+        "authenticationType": "certificateAuthority",
+        "x509Thumbprint": {
+          "primaryThumbprint": null,
+          "secondaryThumbprint": null
+        },
+        "tags": {
+          "deviceType": "Type1, Type2, Type3"
+        },
+        "properties": {
+          "desired": {
+            "prop2": 5
+          }
+        },
+        "capabilities": {
+          "iotEdge": false
+        }
+      }
+    ],
+    "children": {
+      "edge-parent1": [
+        "device1-child",
+        "edge-parent2"
+      ],
+      "edge-parent2": [
+        "device2-child"
+      ],
+      "device2": [
+        "edge-parent1",
+        "simDevice"
+      ]
+    }
+  },
   "modules": {
     "device1-child": [],
+    "edge-parent1": [],
```

Comment:
Is this a latest snapshot of the sample json? There still appears to be a repeat of device identities (where we discussed a module identity object already has the device Id as part of it).

Reply:
This comment has not been resolved yet, I'm still seeing deviceId as the key to a module array